resources/views/community/live.blade.php

```diff
@@ -16,3 +16,73 @@
     along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
 --}}
 @extends("master")
+
+@section('content')
+
+<div class="osu-layout__section osu-layout__section--full livestreams">
+    <div class="osu-layout__row osu-layout__row--with-gutter">
+        <div class="osu-layout__row--page header-row livestream-header">
+            <div class="wide col-sm-12 livestream-header__container">
+                <div>
+                    <h1 class="livestream-header__headline">{{ trans('livestreams.top-headers.headline') }}</h1>
+                    <p class="livestream-header__description">{{ trans('livestreams.top-headers.description') }}</p>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="osu-layout__row osu-layout__row--with-gutter osu-layout__row--page livestream-page">
+    @if ($featuredStream != null)
```

Comment:
I personally prefer strict comparison (`!==`) especially since php's casting is a bit funny (put nicely).

```
>>> 0 == null
=> true
>>> '0' == null
=> false
>>> '0' == 0
=> true
(conclusion: true == false)
```